tests/unit/type/set.tcl

```diff
@@ -177,23 +179,23 @@ foreach type {single multiple single_multiple} {
 
     test "Set encoding after DEBUG RELOAD" {
         r del myintset
-        r del myhashset
+        r del myhashtable
```

Comment:
Here, hashset just means a set backed by a hashtable. It's not new. This name was used also for the dict-backed set.

```suggestion
        r del myhashset
```